docs/sources/tempo/metrics-generator/_index.md

```diff
@@ -46,3 +46,9 @@ Dimensions can be the service name, the operation, the span kind, the status cod
 The more dimensions are enabled, the higher the cardinality of the generated metrics.
 
 To learn more about this processor, read the [documentation]({{< relref "span_metrics/" >}}).
+
+### Exporting metrics
```

Comment:
Can we mention the internal that we use to send is configurable also?

Reply:
Sorry typo.  The `interval` is what I meant, like how often we write metrics.

Reply:
Done!